<launch>

	<!-- Run the Hokuyo laser scanner node -->
	<node name="hokuyo" pkg="hokuyo_node" type="hokuyo_node">
	<param name="min_ang" value="-2.095" />
	<param name="max_ang" value="2.095" />
	<param name="cluster" value="1" />
	<param name="skip" value="1" />
	<param name="hokuyo_node/calibrate_time" value="true" />
	<param name="frame_id" value="base_laser" />
	<remap from="/scan" to="base_scan" />
	<!-- <param name="port" value="/dev/ttyUSB1"/> -->
</node>

<!-- Add to your launch file -->
      <node name="rear_hokuyo" pkg="hokuyo_node" type="hokuyo_node">
    	<param name="port" value="/dev/ttyACM1"/>  <!-- Adjust port -->
    	<param name="frame_id" value="rear_laser"/>
    	<param name="min_ang" value="-2.0944"/>    <!-- -120 degrees -->
    	<param name="max_ang" value="2.0944"/>     <!-- +120 degrees -->
    	<remap from="/scan" to="rear_scan"/>
      </node>

</launch>

